### PR TITLE
Clean up the README post-skeletonization

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request fixes some things I missed in #125 as well as some minor cleanup.

## 💭 Motivation and context ##

The installation/running directions were not updated appropriately with the changes in #125. Since I already needed to fix that portion of the README I did a quick skim of the rest while I was working on the branch.

## 🧪 Testing ##

Automated tests pass. I confirmed the README renders as expected when viewing the branch.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -83,12 +70,13 @@ output will always be written to disk, defaulting to `results.csv`.
...
-  --starttls                  Only check mx records and STARTTLS support.
+  --mx                        Only check MX records
+  --starttls                  Only check MX records and STARTTLS support.
                               (Implies --mx.)
-  --spf                       Only check spf records
-  --dmarc                     Only check dmarc records
-  --debug                     Output should include error messages.
+  --spf                       Only check SPF records
+  --dmarc                     Only check DMARC records
+  --json                      Output is in json format (default csv)
```
Comment (dav3r) [inline README.md]: In for a penny, in for a pound:
```suggestion
  --json                      Output is in JSON format (default CSV)
```

**Review by dav3r — APPROVED** [review]
👍 👍

File: README.md (hunk 1 above)
Comment (mcdonnnj) [reply README.md]: This was actually copied from:
 https://github.com/cisagov/trustymail/blob/3d45f40e9686563bb374570b281a520abbb9ee00/src/trustymail/cli.py#L9-L45
 I'll make the change in both so it's still consistent.

Comment (mcdonnnj) [reply README.md]: Please see [e6d1098](https://github.com/cisagov/trustymail/pull/127/commits/e6d1098099784d6a57d112d1df51e80c43d989f0).

**Review by dav3r — APPROVED** [review]
👍

**Review by jsf9k — APPROVED** [review]

**Pull request merged**